From 8d104f169f0e9a23f86e18d1d8d58b9dd0aa3787 Mon Sep 17 00:00:00 2001
From: Yaoming Qin <b10945@freescale.com>
Date: Wed, 30 Dec 2009 13:20:06 -0500
Subject: [PATCH 012/635] [ENGR00119732] Enable the audio recording on Eclair
 Make the alsa driver work in the same configure
 (sampleRate, channel, format ) in capture as that
 in playback. Make the audio samplerate & channel
 convert work for a bigger range samplerate
 conversion. Fix some bugs in audio recording in
 Android. Signed-off-by: Yaoming
 Qin<B10945@freescale.com>

---
 modules/alsa/alsa_imx51.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)
 mode change 100644 => 100755 modules/alsa/alsa_imx51.cpp

diff --git a/modules/alsa/alsa_imx51.cpp b/modules/alsa/alsa_imx51.cpp
old mode 100644
new mode 100755
index a343f84..2f2b357
--- a/modules/alsa/alsa_imx51.cpp
+++ b/modules/alsa/alsa_imx51.cpp
@@ -133,8 +133,8 @@ static alsa_handle_t _defaults[] = {
         curMode     : 0,
         handle      : 0,
         format      : SND_PCM_FORMAT_S16_LE, // AudioSystem::PCM_16_BIT
-        channels    : 1,
-        sampleRate  : AudioRecord::DEFAULT_SAMPLE_RATE,
+        channels    : 2,
+        sampleRate  : DEFAULT_SAMPLE_RATE,
         latency     : 250000, // Desired Delay in usec
         bufferSize  : 2048, // Desired Number of samples
         modPrivate  : (void *)&setDefaultControls,
-- 
1.8.0

